pkg/handlers/handlers.go

```diff
@@ -123,29 +165,86 @@ const (
 type mimeType string
 
 const (
-	rpmMime    mimeType = "application/x-rpm"
-	cpioMime   mimeType = "application/cpio"
-	unixArMime mimeType = "application/x-unix-archive"
-	arMime     mimeType = "application/x-archive"
-	debMime    mimeType = "application/vnd.debian.binary-package"
+	rpmMime      mimeType = "application/x-rpm"
+	cpioMime     mimeType = "application/cpio"
+	unixArMime   mimeType = "application/x-unix-archive"
+	arMime       mimeType = "application/x-archive"
+	debMime      mimeType = "application/vnd.debian.binary-package"
+	textMime     mimeType = "text/plain; charset=utf-8"
+	xmlMime      mimeType = "text/xml"
+	jsonMime     mimeType = "application/json"
+	csvMime      mimeType = "text/csv"
+	tsvMime      mimeType = "text/tab-separated-values"
+	geoJSONMine  mimeType = "application/vnd.geo+json"
+	ndjsonMime   mimeType = "application/x-ndjson"
+	htmlMime     mimeType = "text/html"
+	phpTextMime  mimeType = "text/x-php"
+	rtfTextMime  mimeType = "text/rtf"
+	jsAppMime    mimeType = "application/javascript"
+	jsTextMime   mimeType = "text/javascript"
+	jsMime       mimeType = "application/x-javascript"
+	srtMime      mimeType = "application/x-subrip"
+	srtXMime     mimeType = "application/x-srt"
+	srtTextMime  mimeType = "text/x-srt"
+	vttMime      mimeType = "text/vtt"
+	luaMime      mimeType = "text/x-lua"
+	perlMime     mimeType = "text/x-perl"
+	pythonMime   mimeType = "text/x-python"
+	pyAppMime    mimeType = "application/x-python"
+	pyScriptMime mimeType = "application/x-script.python"
+	tclTextMime  mimeType = "text/x-tcl"
+	tclMime      mimeType = "application/x-tcl"
 )
 
-// selectHandler dynamically selects and configures a FileHandler based on the provided fileReader.
+// textBasedOrUnsupportedMimeTypes is a set of MIME types that are text-based or unsupported.
+var textBasedOrUnsupportedMimeTypes = map[mimeType]struct{}{
```

Comment:
This comment could make it clearer why "text-based" and "unsupported" are grouped together. 

e.g., What does unsupported mean? Will TruffleHog not scan these?